modules/drivers/mongo/test/metabase/driver/mongo/parameters_test.clj

```diff
@@ -179,10 +179,10 @@
       (doseq [[operator form input] [[:number/<= {"price" {"$lte" 42}} [42]]
                                      [:number/>= {"price" {"$gte" 42}} [42]]
                                      [:number/!= {"price" {"$ne" 42}} [42]]
-                                     [:number/= {"price" {"$eq" 42}} [42]]
-                                     ;; i don't understand the $ on price here
-                                     [:number/between {"$and" [{"$expr" {"$gte" ["$price" 42]}}
-                                                               {"$expr" {"$lte" ["$price" 142]}}]} [42 142]]]]
+                                     [:number/= {"price" 42} [42]]
```

Comment:
I think this really hammers home my confusion about this. It seems like these should all be the same but why is `$eq` behaving differently.

Reply:
`{price 4}` and `{price {$eq 4}}` both mean the exact same thing and give you the same results. The former is more idiomatic tho from my understanding -- all of the examples from the MongoDB website use the former syntax for simple equality and only use the latter with an operator other than `$eq`

Previously we have been generating `{$expr {$eq [$price 4]}}`, which returns the same results if `$price` is an atomic value but not if it's an array. So it doesn't make a difference _here_ but it was causing the bug I ended up fixing